tools/phenomenal/ecomet/ecomet_11_species_nmds.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--Proposed Tool Section: [Eco-Metabolomics]-->
+<tool id="ecomet_species_nmds" name="Species NMDS" version="1.0">
+  <description>Perform NMDS on species</description>
+  <stdio>
+    <regex match="" source="stderr" level="warning" description="R messages" />
+    <exit_code range="1:" level="fatal" description="Tool error" />
+  </stdio>
+  <command><![CDATA[
+mtbls520_11_species_nmds.r $infile1 $outfile1 $outfile2 $outfile3;
+  ]]>
+  </command>
+  <inputs>
+    <param name="infile1" type="data" format="rdata" optional="False" label="Input Rdata file" />
+  </inputs>
+  <outputs>
+    <data name="outfile1" type="data" format="pdf" label="Species NMDS Stress" />
+    <data name="outfile2" type="data" format="pdf" label="Species NMDS Procrustes" />
+    <data name="outfile3" type="data" format="pdf" label="Species NMDS" />
+  </outputs>
+  <help><![CDATA[
+This is the MTBLS520 study.
+The study will be published soon. This code is work-in-progress. Do not redistribute unless study published.
```

Comment:
@korseby:  given the message here, are you sure you want to include this tool in PhenoMeNal at this time?